docs/course_materials/exercises/03_Variables_and_datatypes/homework.md

```diff
@@ -0,0 +1,88 @@
+# Variables and Data Types - HomeWork exercise 
+
+## Variables, numbers and boolean 
+
+Before answering the questions:
+1. Change Directory to lesson2 
+2. Create a directory called Homework2 
+3. Create a file called variables.py and answer the following:
+
+### variables.py - Exercise 1
+Add two variables together:
+1. Declare three variables (x, y, and z) as integers.
+2. Assign a value of 9 to x
+3. Assign a value of 7 to y
+4. Make z equal to the sum of those two variables. 
+5. Print out the contents of the z variable.
+
+### variables.py - Exercise 2
+Create a new file called calculations.txt And answer the following questions:
+
+In order for you to answer the questions, please use python, in the terminal and run the commands you need. 
```

Comment:
For answering the following questions, please use the python terminal and run the commands you need.